libraries/botframework-connector/src/teams/operations/teams.ts

```diff
@@ -250,18 +250,18 @@ export class Teams {
      */
     sendMeetingNotification(
         meetingId: string,
-        notification: TeamsMeetingNotification,
+        notification: MeetingNotification,
         options?: msRest.RequestOptionsBase,
-    ): Promise<Models.TeamsSendMeetingNotificationResponse>
+    ): Promise<Models.TeamsMeetingNotificationResponse>
```

Comment:
Is there any reason why we didn't remove `Teams `for `TeamsMeetingNotificationResponse`

Reply:
I described the reason in the comment above